api/src/main/java/org/openmrs/BaseChangeableOpenmrsData.java

```diff
@@ -9,45 +9,53 @@
  */
 package org.openmrs;
 
+import javax.persistence.Column;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.MappedSuperclass;
 import java.util.Date;
 
 /**
  * Base superclass for all mutable OpenmrsData.
- * 
+ *
  * @since 2.2
  */
+@MappedSuperclass
 public abstract class BaseChangeableOpenmrsData extends BaseOpenmrsData {
+
+	@ManyToOne
+	@JoinColumn(name = "changed_by")
+	private User changedBy;
+
+	@Column(name = "date_changed")
+	private Date dateChanged;
 	
 	/**
 	 * @see Auditable#getChangedBy()
 	 */
 	@Override
 	public User getChangedBy() {
-		return super.getChangedBy();
+		return this.changedBy;
 	}
 	
 	/**
 	 * @see Auditable#setChangedBy(User)
 	 */
 	@Override
 	public void setChangedBy(User changedBy) {
-		super.setChangedBy(changedBy);
+		this.changedBy = changedBy;
 	}
 	
 	/**
 	 * @see Auditable#getDateChanged()
 	 */
 	@Override
-	public Date getDateChanged() {
-		return super.getDateChanged();
-	}
+	public Date getDateChanged() {return this.dateChanged;}
```

Comment:
no need for 'this' here as well and other similar places where it was used in getters